pipelines/tekton/aiedge-e2e/tasks/kserve-download-model.yaml

```diff
@@ -17,13 +17,16 @@ spec:
     description: The S3 URL used to download the model
   steps:
   - name: download-model-s3
-    image: quay.io/opendatahub/kserve-storage-initializer:v0.11
+    image: quay.io/opendatahub/kserve-storage-initializer:v0.11.1.3 
     script: |
       mkdir -p $(workspaces.workspace.path)/model_dir-$(params.model-name)/
 
       export S3_URL="s3://$(params.s3-bucket-name)/$(params.model-name)"
       echo -n $S3_URL | tee $(results.s3-url.path) ;
 
+      export CA_BUNDLE_CONFIGMAP_NAME=ssl-ca-directory
+      export AWS_CA_BUNDLE=$(workspaces.ssl-ca-directory.path)/cabundle.crt
```

Comment:
We are using `cabundle.crt` here but the git-clone task defaults to `ca-bundle.crt`  We should have some parity between the file name between this task and the git-clone task assuming that the value of `AWS_CA_BUNDLE` is the actual CA bundle filename we are mounting to the file

Reply:
Yes, this was a temperaory change because depending on how you use kserve you need that exact name but these have now been seperated out to s3 and git and the names are the same as it doens't affect kserve